lib/services/composer.js

```diff
@@ -71,6 +120,8 @@ function filterBaseInstanceReferences(obj) {
 module.exports.resolveComponentReferences = resolveComponentReferences;
 module.exports.composePage = composePage;
 module.exports.filterBaseInstanceReferences = filterBaseInstanceReferences;
+module.exports.referenceProperty = referenceProperty;
```

Comment:
had to export this to use it for calling `resolveComponentReferences` in order to remove the breaking change for that call signature